esp/xslt/esdl2ecl.xslt

```diff
@@ -229,6 +229,7 @@ end;
 		<xsl:when test="$basic_type='int'"><xsl:text disable-output-escaping="yes">integer</xsl:text><xsl:if test="not(@ecl_max_len)"><xsl:value-of select="$size"/></xsl:if></xsl:when>
 		<xsl:when test="$basic_type='long'"><xsl:text disable-output-escaping="yes">integer4</xsl:text><xsl:if test="not(@ecl_max_len)"><xsl:value-of select="$size"/></xsl:if></xsl:when>
 		<xsl:when test="$basic_type='short'"><xsl:text disable-output-escaping="yes">integer2</xsl:text></xsl:when>
+        <xsl:when test="$basic_type='int64'"><xsl:text disable-output-escaping="yes">integer8</xsl:text></xsl:when>
```

Comment:
Indentation looks off.

Reply:
it is, I used spaces but the file uses tabs. I set up my editor to automatically convert new tabs to spaces. Should I force tabs in order to maintain the current format?